File: CooCooBase/Models/Core Data/LoyaltyCapped+CoreDataProperties.m
//
//  LoyaltyCapped+CoreDataProperties.m
//  
//
//  Created by Ravi Jampala on 3/28/18.
//
//

#import "LoyaltyCapped+CoreDataProperties.h"

@implementation LoyaltyCapped (CoreDataProperties)

+ (NSFetchRequest<LoyaltyCapped *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"LoyaltyCapped"];
}

@dynamic rideCount;
@dynamic ticketId;
@dynamic activatedTime;
@dynamic walletId;
@dynamic productId;
@dynamic referenceActivatedTime;
@dynamic productName;
@end

File: Genfare/ManagedObjects/LoyaltyCapped+CoreDataProperties.swift
//
//  LoyaltyCapped+CoreDataProperties.swift
//  Genfare
//
//  Created by vishnu on 17/04/19.
//  Copyright © 2019 Omniwyse. All rights reserved.
//
//

import Foundation
import CoreData


extension LoyaltyCapped {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<LoyaltyCapped> {
        return NSFetchRequest<LoyaltyCapped>(entityName: "LoyaltyCapped")
    }

    @NSManaged public var activatedTime: NSNumber?
    @NSManaged public var productId: String?
    @NSManaged public var productName: String?
    @NSManaged public var referenceActivatedTime: NSNumber?
    @NSManaged public var rideCount: NSNumber?
    @NSManaged public var ticketId: NSNumber?
    @NSManaged public var walletId: String?

}
